### ISSUE TITLE
CRNN model bad precision.

### ISSUE THREAD

**Issue by cosimo17** [openvinotoolkit/openvino#298]
when I convert CRNN model from tensorflow to IR and run it on NCS2,I got a very bad accuracy.
I have checked the data distribution(mean,std,max,min) between output given by tensorflow model and output given by NCS2. I find there is large gap between these two outputs.
Is there any one meet the same problem? Or is there any one know how to solve this problem?
Thanks.

**Comment by elvin-nnov** [comment]
Can you verify the results on CPU to narrow down the problematic place in OpenVINO software stack? Could you try FP16 and FP32 IR?

**Comment by cosimo17** [comment]
I made a low-level mistake during preprocess input image.I have fixed it.Thanks.
But I have another problem.I have a CRNN model trainer by tensorflow. I convert it to IR model.
I run it on PC+NCS2,it is OK. but I got error when I run it on raspberry pi+NCS2.
    RuntimeError: The plugin does not support networks with MIXED format.
Then I checked the xml file.I find there are FP16 precision and I32 precision.I have point the --data_type FP16 when I run mo_tf.py.
why I got this? How to get a pure FP16 precision model?
Thanks
